include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -270,6 +257,37 @@ class CollectiveEpilogue<
     return fusion_callbacks.is_producer_load_needed();
   }
 
+  template<typename Tensor>
+  CUTLASS_DEVICE auto reshape_with_unit_insertion(Tensor&& tensor) {
+    using namespace cute;
+    
+    auto orig_layout = tensor.layout();
+    auto orig_shape = orig_layout.shape();
+    auto orig_stride = orig_layout.stride();
+    
+    auto first_dim = get<0>(orig_shape);
+    auto outer_part = get<0>(first_dim); 
+    auto inner_part = get<1>(first_dim);
+    
+    auto first_stride = get<0>(orig_stride); 
+    auto outer_stride = get<0>(first_stride); 
+    auto inner_stride = get<1>(first_stride); 
+    
+    auto target_shape = make_shape(
+        make_shape(outer_part, _1{}), 
+        get<0>(inner_part),              
+        get<1>(inner_part)  
+    );
+    
+    auto target_stride = make_stride(
+        make_stride(outer_stride, _0{}),  
+        get<0>(inner_stride), 
+        get<1>(inner_stride) 
+    );
+    
+    return make_tensor(tensor.data(), make_layout(target_shape, target_stride));
```

Comment:
Thanks for the revision! 

IMHO, this seems to be a very good workaround to leverage new C, D copy atoms, if it'd work for any copy atoms used in epilogues.